src/services/browser.js

```diff
@@ -0,0 +1,279 @@
+/** Copied from: https://github.com/videojs/video.js/blob/main/src/js/utils/browser.js */
+
+/**
+ * Whether or not this device is an iPod.
+ *
+ * @static
+ * @type {Boolean}
+ */
+export let IS_IPOD = false;
+
+/**
+ * The detected iOS version - or `null`.
+ *
+ * @static
+ * @type {string|null}
+ */
+export let IOS_VERSION = null;
+
+/**
+ * Whether or not this is an Android device.
+ *
+ * @static
+ * @type {Boolean}
+ */
+export let IS_ANDROID = false;
+
+/**
+ * The detected Android version - or `null` if not Android or indeterminable.
+ *
+ * @static
+ * @type {number|string|null}
+ */
+export let ANDROID_VERSION;
```

Comment:
For consistency should this be initialized to `null`?
```suggestion
export let ANDROID_VERSION = null;
```

Reply:
Absolutely! 💯